GoogleForCreators/web-stories-wp
# Media Library: Hide edit/delete options if lacking permissions

## Context



## Summary

Users with the "Contributor" role are shown options to edit and delete items in the media library on the left hand side, but because they don't actually have permissions to do that it doesn't work of course.

We should hide the "..." menu if the user does not have permission to edit media.

## Relevant Technical Choices



## To-do



## User-facing changes

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Login as contributor. 
2. Create a new story. 
3. Hover over a media library element like image / video. 
4. Do not see `...` menu (aka it's not possible to edit the media) 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8221
